content_delegate.js

```diff
@@ -240,13 +236,23 @@ ContentDelegate.prototype.onDocumentViewSubmit = function (event) {
     let image_string = $('td:contains(Image)').text();
     let regex = /(\d+)-(\d+)/;
     let matches = regex.exec(image_string);
+    if (!matches) {
+      // This happens on bankruptcy claim pages, which are a form of document in
+      // CM/ECF that we don't want. In this case, just submit the original form.
+      form.submit();
```

Comment:
Huh. Yeah. Interesting that it worked regardless.

Reply:
Well, if you submit a form twice, it'll "work." (Maybe you get charged twice? Although I've always wondered if PACER is forgiving if you download the same PDF 2x in a row, e.g., because network problems interrupted the first attempt).

Reply:
Well, what's weird is the next line is this: 

    document_number = matches[1];

Which would have crashed things if it was reached, since that's where we began this bug.